### PR TITLE
Jason merge copy paste

### PR DESCRIPTION
**Pull request by Topia99**
1. Share itinerary
2. display activities in List{} view, allow long press to order activities.
3. Move trip to top of the trip list if edit.

### CODE REVIEW

File: Travel-Planner/Travel-Planner/Screens/AddTripView.swift [hunk 1]
```diff
@@ -22,10 +22,12 @@ struct AddTripView: View {
...
                 TextField("Title", text: $newTripTitle)
             }
-            
+        
             Section(header: Text("Date")) {
-                DatePicker("Start Date", selection: $startDate, displayedComponents: [.date])
-                DatePicker("End Date", selection: $endDate, in: startDate..., displayedComponents: [.date])
+                DatePicker("Start Date", selection: $startDate)
+                DatePicker("End Date", selection: $endDate, in: startDate...)
+//                DatePicker("Start Date", selection: $startDate, displayedComponents: [.date])
```
Comment (Topia99) [inline Travel-Planner/Travel-Planner/Screens/AddTripView.swift]: Use comment version

File: Travel-Planner/Travel-Planner/Utilities/DateUtils.swift [hunk 2]
```diff
@@ -25,6 +25,8 @@ struct DateUtils {
     }
     
     static func numberOfDaysBetweenTwoDates(_ startDate: Date, _ endDate: Date) -> Int {
+        print(startDate)
```
Comment (Topia99) [inline Travel-Planner/Travel-Planner/Utilities/DateUtils.swift]: remove

File: Travel-Planner/Travel-Planner/Views/DayPlanView/DayPlan/DayPlanView.swift [hunk 3]
```diff
@@ -11,27 +11,33 @@ struct DayPlanView: View {
     
     @StateObject var vm: ActivityViewModel
     @State var isShowingAddActivity: Bool = false // Controls the presentation of the TripItemDetail sheet.
+    @State private var selectedActivity: ActivityEntity? = nil
+    
+    struct Item: Identifiable {
+        let id: UUID
```
Comment (Topia99) [inline Travel-Planner/Travel-Planner/Views/DayPlanView/DayPlan/DayPlanView.swift]: Remove Item. Only for testing

**Review by Topia99 — COMMENTED** [review]
Code need some clean up

**Review by Topia99 — COMMENTED** [review]
Ready for merge

**Review by JasonZeng9912 — APPROVED** [review]

**Pull request merged**